script.js

```diff
@@ -84,11 +102,15 @@ taskForm.onsubmit = async (e) => {
     });
 
     const result = await response.json();
+    // showSubmitLoader(false)
 
     alert(result.message);
   } catch (error) {
+    // showSubmitLoader(false)
     alert(`Error: ${error}`);
   }
+    showSubmitLoader(false)
+
```

Comment:
Can we make use of try, catch, finally block?

Reply:
sure. I'll make changes